.pre-commit-config.yaml

```diff
@@ -16,30 +18,36 @@ repos:
     -   id: reorder-python-imports
         files: ^src/|tests/
 
--   repo: local
+-   repo: https://github.com/pre-commit/mirrors-mypy.git
+    rev: v0.910
     hooks:
     -   id: mypy
-        name: mypy
```

Comment:
I think we ought to keep these lines and drop the new `pass_filenames`. Maybe some copy/paste overloading from `doi-service` occurred?